.github/workflows/active_model_otp.yml

```
@@ -0,0 +1,43 @@
+name: Active Model OTP
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
```

I think something similar to this will make the magic too https://github.com/robertomiranda/delegated_type/blob/107845dada92dd048c275f56eac9d22251df3856/.github/workflows/tests.yml#L10-L14 

```suggestion
on:
  push:
    branches: [main]
  pull_request:
```

My understanding is that `[opened, synchronize, reopened, edited]` this one is the default value https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request

---

The default value is [opened, synchronize, reopened]. I added the "edited" for situations when new commits are added in PR after open it (just like this PR, for example) 👍  in this case, it was need to define the events to be captured by github actions